scripts/TW01-JP/c100211053.lua

```diff
@@ -0,0 +1,117 @@
+--レアル・ジェネクス・チューリング
+--Scripted by zengsxing
+local s,id,o=GetID()
+function s.initial_effect(c)
+	--spsummon
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(id,0))
+	e1:SetCategory(CATEGORY_SPECIAL_SUMMON)
+	e1:SetType(EFFECT_TYPE_QUICK_O)
+	e1:SetProperty(EFFECT_FLAG_CARD_TARGET)
+	e1:SetRange(LOCATION_HAND)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetCountLimit(1,id)
+	e1:SetHintTiming(0,TIMINGS_CHECK_MONSTER+TIMING_MAIN_END)
+	e1:SetCondition(s.spcon)
+	e1:SetTarget(s.sptg)
+	e1:SetOperation(s.spop)
+	c:RegisterEffect(e1)
+	--synchro level
+	local e2=Effect.CreateEffect(c)
+	e2:SetType(EFFECT_TYPE_SINGLE)
+	e2:SetCode(EFFECT_SYNCHRO_MATERIAL_CUSTOM)
+	e2:SetTarget(s.syntg)
+	e2:SetValue(1)
+	e2:SetOperation(s.synop)
+	c:RegisterEffect(e2)
+	local e3=Effect.CreateEffect(c)
+	e3:SetType(EFFECT_TYPE_SINGLE)
+	e3:SetProperty(EFFECT_FLAG_SINGLE_RANGE)
+	e3:SetCode(id)
+	e3:SetRange(LOCATION_MZONE)
+	c:RegisterEffect(e3)
+end
+function s.cardiansynlevel(c,sc)
+	if c:IsHasEffect(id) and sc:IsSetCard(0x2) then
+		return 3+(1<<16)
+	else
+		return c:GetSynchroLevel(sc)
+	end
+end
+function s.synfilter(c,syncard,tuner,f)
+	return c:IsFaceupEx() and c:IsCanBeSynchroMaterial(syncard,tuner) and (f==nil or f(c,syncard))
+end
+function s.syncheck(c,g,mg,tp,lv,syncard,minc,maxc)
+	g:AddCard(c)
+	local ct=g:GetCount()
+	local res=s.syngoal(g,tp,lv,syncard,minc,ct)
+		or (ct<maxc and mg:IsExists(s.syncheck,1,g,g,mg,tp,lv,syncard,minc,maxc))
+	g:RemoveCard(c)
+	return res
+end
+function s.syngoal(g,tp,lv,syncard,minc,ct)
+	return ct>=minc and Duel.GetLocationCountFromEx(tp,tp,g,syncard)>0
+		and (g:CheckWithSumEqual(Card.GetSynchroLevel,lv,ct,ct,syncard)
+			or g:CheckWithSumEqual(s.cardiansynlevel,lv,ct,ct,syncard))
+		and aux.MustMaterialCheck(g,tp,EFFECT_MUST_BE_SMATERIAL)
+end
+function s.syntg(e,syncard,f,min,max)
+	local minc=min+1
+	local maxc=max+1
+	local c=e:GetHandler()
+	local tp=syncard:GetControler()
+	local lv=syncard:GetLevel()
+	if lv<=c:GetLevel() and lv<=s.cardiansynlevel(c) then return false end
```

Comment:
redtext on line 35 caused here: `attempt to index a nil value (local 'sc')`
```suggestion
	if lv<=c:GetLevel() and lv<=s.cardiansynlevel(c,syncard) then return false end
```